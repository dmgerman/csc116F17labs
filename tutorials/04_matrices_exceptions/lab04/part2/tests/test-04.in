3 3
1 2 3
4 5 6
7 8 9

4 5
2 3 1 4 5