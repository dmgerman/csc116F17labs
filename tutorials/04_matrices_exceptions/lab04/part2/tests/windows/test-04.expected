Reading first matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 3 x 3
M1: 
1 2 3 
4 5 6 
7 8 9 
Reading second matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 
Error reading matrix: Invalid matrix element
